examples/quill.html

```diff
@@ -42,7 +42,7 @@
       const delta = embed
         ? { insert: JSON.parse(embed), attributes } :
         {
-          insert: change.content || ' ',
+          insert: change.content || '',
```

Comment:
Oh, that was my mistake. There was no fallback handling in the previous source:

https://github.com/yorkie-team/yorkie-js-sdk/blob/de792b3d40c831a832fc53902b5411d6272e97e4/examples/quill.html#L44